hector.md

```diff
@@ -1,14 +1,17 @@
----
+﻿---
 layout: index
-title: Climate Module – Hector	
+title: Earth System Module – Hector v2.0	
 prev: water.html
 next: choice.html
-gcam-version: v4.4
+gcam-version: v5.1
 ---
 
-This section describes the new climate module - Hector - that is available for use in GCAM. MAGICC5.3 (Wiglley, 2008) has traditionally been the only climate module available in GCAM.  In GCAM's recent release, Hector is now the default climate model(Hartin et al., 2015).  Both Hector and MAGICC are reduced-form climate carbon-cycle models. 
+This section describes the carbon-cycle climate module - Hector - that is available for use in GCAM. MAGICC5.3 (Wigley, 2008) has traditionally been the only climate module available in GCAM.  MAGICC5.3 will no longer be supported in this release.
```

Comment:
Can we change the second sentence to say: "While MAGICC5.3 is still included in the GCAM5.1 code base, we will not be supporting this option going forward." or something like that?